gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -352,6 +352,11 @@
             <property name="message" value="Logging with System.err is not allowed because it has no metadata and can't be configured at runtime. Please use an SLF4J logger instead, e.g. log.info(&quot;Message&quot;)."/>
             <property name="ignoreComments" value="true"/>
         </module>
+        <module name="RegexpSinglelineJava">
+            <property name="id" value="PreferInclusiveLanguage"/>
+            <property name="format" value="(whitelist|blacklist|slave)"/>
+            <property name="message" value="Prefer choosing names that don't have racial undertones, e.g. 'blocklist/allowlist' or 'server/client'"/>
```

Comment:
we could provide more suggested alternatives here, e.g. leader/follower, replica, primary/secondary?

Reply:
+1 for additional suggestions.

Some (large) projects exclusively use error-prone instead of checkstyle, I wonder if we could implement this using error-prone and automatically suggest changes inline?

Reply:
Should we also add "master" to the format?

Reply:
I think there are 7658 usages of master in java files internally. I think I'd like to rely on humans to prioritise this themselves, as blocking rollout of baseline is quite painful.